multisite/template/loaders/filesystem.py

```diff
@@ -32,4 +37,15 @@ def get_template_sources(self, template_name, template_dirs=None):
                 # The joined path was located outside of this particular
                 # template_dir (it might be inside another one, so this isn't
                 # fatal).
-                pass
+                continue
+
+            # Template loading was changed in Django 1.9, and
+            # django now expects an Origin object to be returned
+            try:
+                yield Origin(
+                    name=name,
+                    template_name=template_name,
+                    loader=self
+                )
+            except NameError:
```

Comment:
As a stylistic change (since this `NameError` here is a bit obscure to me) how about changing the `Origin` import into

```python
try:
    from django.template import Origin
except ImportError:
    # Django < 1.9
    def Origin(name="", *args, **kwargs):
        return name
```